From 71479f515168dcbe32267475724197aa24559754 Mon Sep 17 00:00:00 2001
From: wangguosheng <wangguosheng@eswincomputing.com>
Date: Wed, 27 Aug 2025 13:18:36 +0800
Subject: [PATCH 483/540] style:fix the warning in DTS

Changelogs:
1. fix the warning in DTS
arch/riscv/boot/dts/eswin/eic7702-deepcomputing-fml13v03-evt.dts:1100.17-1116.4: Warning (spi_bus_bridge): /soc/spi@50814000/spi@0: incorrect #address-cells for SPI bus
  also defined at arch/riscv/boot/dts/eswin/eic7702-deepcomputing-fml13v03-evt.dts:1639.10-1649.3
arch/riscv/boot/dts/eswin/eic7702-deepcomputing-fml13v03-evt.dts:1100.17-1116.4: Warning (spi_bus_bridge): /soc/spi@50814000/spi@0: incorrect #size-cells for SPI bus
  also defined at arch/riscv/boot/dts/eswin/eic7702-deepcomputing-fml13v03-evt.dts:1639.10-1649.3
arch/riscv/boot/dts/eswin/eic7702-deepcomputing-fml13v03-evt.dtb: Warning (spi_bus_reg): Failed prerequisite 'spi_bus_bridge'

Signed-off-by: wangguosheng <wangguosheng@eswincomputing.com>
---
 .../riscv/boot/dts/eswin/eic7702-deepcomputing-fml13v03-evt.dts | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/riscv/boot/dts/eswin/eic7702-deepcomputing-fml13v03-evt.dts b/arch/riscv/boot/dts/eswin/eic7702-deepcomputing-fml13v03-evt.dts
index 7e8270a8393fb..fce5c14cea746 100755
--- a/arch/riscv/boot/dts/eswin/eic7702-deepcomputing-fml13v03-evt.dts
+++ b/arch/riscv/boot/dts/eswin/eic7702-deepcomputing-fml13v03-evt.dts
@@ -1142,7 +1142,7 @@ &ssi1 {
 	cs-gpios = <&portc 24 GPIO_ACTIVE_LOW>;
 	num-cs = <1>;
 
-	fwos_ec: spi@0 {
+	fwos_ec: ec@0 {
 		compatible = "google,cros-ec-spi";
 		reg = <0>;
 		status = "okay";
-- 
2.53.0

